packages/app/src/runner/event-manager.ts

```diff
@@ -396,6 +410,14 @@ export class EventManager {
     // @ts-ignore
     window.Cypress = Cypress
 
+    this.studioStore.setup(config)
+
+    const isDefaultProtocolEnabled = Cypress.config('isProtocolEnabled')
```

Comment:
I wonder if we should name the config value `isDefaultProtocolEnabled` to make it clearer?

Reply:
I'm good with that

Reply:
Renamed: [`e619cb2` (#31239)](https://github.com/cypress-io/cypress/pull/31239/commits/e619cb23c7d692f22ef0e29370b9b5d8555e6a72)